DigitalLearningSolutions.Web/Views/LearningMenu/PostLearning/PostLearning.cshtml

```diff
@@ -44,7 +44,7 @@
   </div>
 </div>
 
-<partial name="Shared/_AssessmentPageImportantInformation" />
+<partial name="PostLearning/_ImportantInformation" />
```

Comment:
Minor but as discussed on slack I think it would make sense to add `PostLearning` to the partial name. Just because having two files with the same name can get confusing 🙂